src/Symfony/Component/Form/ChoiceList/Factory/CachingFactoryDecorator.php

```diff
@@ -145,8 +145,9 @@ public function createListFromLoader(ChoiceLoaderInterface $loader, mixed $value
         return $this->lists[$hash];
     }
 
-    public function createView(ChoiceListInterface $list, mixed $preferredChoices = null, mixed $label = null, mixed $index = null, mixed $groupBy = null, mixed $attr = null, mixed $labelTranslationParameters = []): ChoiceListView
+    public function createView(ChoiceListInterface $list, mixed $preferredChoices = null, mixed $label = null, mixed $index = null, mixed $groupBy = null, mixed $attr = null, mixed $labelTranslationParameters = []/* , bool $duplicatePreferredChoices = true */): ChoiceListView
```

Comment:
```suggestion
    /**
     * @param bool $duplicatePreferredChoices
     */
    public function createView(ChoiceListInterface $list, mixed $preferredChoices = null, mixed $label = null, mixed $index = null, mixed $groupBy = null, mixed $attr = null, mixed $labelTranslationParameters = []/* , bool $duplicatePreferredChoices = true */): ChoiceListView
```

please add similar annotations to announce the new argument everywhere it's added (public methods only of course)

Reply:
Thanks for your review!
I added the annotations and added some comments to describe the new agument in the interface.